.idea/modules.xml

```diff
@@ -3,6 +3,7 @@
   <component name="ProjectModuleManager">
     <modules>
       <module fileurl="file://$PROJECT_DIR$/.idea/nest-web.iml" filepath="$PROJECT_DIR$/.idea/nest-web.iml" />
+      <module fileurl="file://$PROJECT_DIR$/.idea/nest-web.iml" filepath="$PROJECT_DIR$/.idea/nest-web.iml" />
```

Comment:
Why this file required?

What is the PR name, next time raise PR subject as karthik_subscription_30Dec2019